engine/crates/engine-v2/src/engine/execute/prepare.rs

```diff
@@ -17,9 +18,10 @@ impl<'ctx, R: Runtime> PreExecutionContext<'ctx, R> {
     pub(super) async fn prepare_operation(
         &mut self,
         request: Request,
+        operation_info: &mut OperationBuilder,
```

Comment:
Would name those `OperationHookInfoBuilder` or smth like this imho. `OperationBuilder` is quite ambiguous here